kili/cli.py

```diff
@@ -21,17 +21,26 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 api_key_option = click.option(
-    '--api-key', type=str, envvar='KILI_API_KEY', required=True,
+    '--api-key', type=str, default=None,
     help=(
-        'Your Kili API key (overrides the KILI_API_KEY environment variable). '
-        'If not passed, requires the KILI_API_KEY environment variable to be set.'
+        'Your Kili API key (overrides the "KILI_API_KEY" environment variable). '
+        'If not passed, requires the "KILI_API_KEY" environment variable to be set.'
+    ),
+    show_default=(
+        'current user ("KILI_API_KEY" environment variable)'
     )
 )
 
 endpoint_option = click.option(
-    '--endpoint', type=str,
-    default='https://cloud.kili-technology.com/api/label/v2/graphql',
-    help='The API Endpoint.'
+    '--endpoint', type=str, default=None,
+    help=(
+        'API Endpoint (overrides the "KILI_API_ENDPOINT" environment variable). '
+        'If not passed, If not passed, default to Kili SaaS: '
```

Comment:
remove  'If not passed' (same reason as above)

Reply:
I suggest : 
`api_key_option = click.option(
    '--api-key', type=str, default=None,
    help=(
        'Your Kili API key. '
    ),
    show_default=(
        '"KILI_API_KEY" environment variable'
    )
)

endpoint_option = click.option(
    '--endpoint', type=str, default=None,
    help=(
        'API Endpoint. '
    ),
    show_default=(
        '"KILI_API_KEY" environment variable,'
        ' Kili SAAS: "https://cloud.kili-technology.com/api/label/v2/graphql"'
    )
)`